pyproject.toml

```diff
@@ -61,6 +61,7 @@ dependencies = [
   "arize-phoenix-evals>=0.13.1",
   "fastapi",
   "pydantic>=1.0,!=2.0.*,<3,", # exclude 2.0.* since it does not support the `json_encoders` configuration setting
+  "pyjwt",
```

Comment:
@axiomofjoy not 100% sure on the authlib dependency yet and so just minimizing the dependencies for now. I think it's pretty interchangeable if needed.

Reply:
sg